salt/api-gateway/config/lib-systemd-system-kong.service

```diff
@@ -7,7 +7,7 @@ WantedBy=multi-user.target
 
 [Service]
 StandardError=syslog
-ExecStart=/usr/local/bin/kong start --nginx-conf /etc/kong/nginx.lua
```

Comment:
how come a systemd configuration was already there in 14.04?

Reply:
https://github.com/elifesciences/api-gateway-formula/pull/5

this is not my first attempt at upgrading our projects to 16.04